webapp/src/ts/modals/mm-modal/mm-modal.ts

```diff
@@ -20,7 +20,7 @@ export class MmModal {
   @Input() titleKey;
   @Input() submitKey;
   @Input() submittingKey;
-  @Input() cancelKey;
+  @Input() cancelKey = 'Cancel';
```

Comment:
I fear setting this default here will  make things harder to track. 

I think it might be clearer if we revert this and change the template so it checks if `cancelKey` is explicitly `false`. If `cancelKey` is explicitly false, it means we don't want a cancel button. If the `cancelKey` is another falsy value, fallback to the default 'Cancel' in the template, so we know what to expect from looking at a single file. What do you think?